egs2/thchs30/tts1/local/thchs-30_data_prep.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
If phoneme is provided, why don't you make two data directories?
- train/dev/test: standard text
- train_phn/dev_phn/test_phn: phoneme_text

Then, we can select in `run.sh`.